config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
-  get 'pages/home'
+  # get 'pages/home'
```

Comment:
Please, do not remove this line, it is needed for `root to:` routine on line 11.